grisera_api/graph_api_config.py

```diff
@@ -2,5 +2,5 @@
 import os
 
 graph_api_host = os.environ.get('GRAPH_API_HOST') or 'localhost'
-graph_api_port = 18080
+graph_api_port = 8000
```

Comment:
I think you didn't mean to commit it 😁

Reply:
This 18080 is meant for the _Docker compose_. Maybe we should use an appropriate environment variable?
`graph_api_port = os.environ.get('GRAPH_API_PORT') or '8000'`

and in  `grisera-framework/docker-compose.yml` modify `grisera_api` container definition:
```
  grisera_api:
    build: grisera_api
    ports:
      - "18081:80"
    extra_hosts: 
      - "host.docker.internal:host-gateway"
    environment:
      - GRAPH_API_HOST=host.docker.internal
      - GRAPH_API_PORT=18080
```
WDYT?

Reply:
It could be useful solution